pkg/timer/timerSync.go

```diff
@@ -55,7 +55,7 @@ func (ws *TimerSync) syncSvc() {
 			}
 			ws.logger.Fatal("failed to get time trigger list", zap.Error(err))
 		}
-		ws.timer.Sync(triggers.Items)
+		ws.timer.Sync(triggers.Items) //nolint: errCheck
```

Comment:
I don't think `nolint` is a good idea. But still nice work.

Reply:
> I check the commits, warnings still. Most of them are cleaned.

The PR only fixes `golanci-lint` warnings in `pkg`. Once this PR is merged I will send another PR with fixes across the repo and that also adds `golanci-lint` to the CI. 

There are plenty of warnings across the repo from `go-lint` that also needs to be addressed to further improve the code quality. And I will be happy to take them next as I contribute further. :slightly_smiling_face: 🚀